pkg/eval/platform/platform.go

```diff
@@ -35,9 +38,41 @@ import (
 // Whether or not the platform is Microsoft Windows.
 // This is read-only.
 
+//elvdoc:fn hostname &strip-domain=false
+//
+// The host name of the system. If option `&strip-domain` is used any domain
```

Comment:
Function docs should start with a verb, so this should say "Outputs the host name of the system".